__tests__/__snapshots__/App-test.js.snap

```diff
@@ -215,6 +215,7 @@ exports[`Button Example Page 1`] = `
       <View>
         <View>
           <Text
```

Comment:
Not necessary for this change being made on a timeline, but given how much this is duplicated I think a styled helper component is in order:

```js
<Header text="A simple Button."/>
```

And somewhere else co-defined:
```js
function Header({text: string}) {
  return (
    <Text            
      accessibilityRole="header"
      style={
        {
          "color": "rgb(28, 28, 30)",
          "fontSize": 20,
          "marginBottom": 10,
          "marginTop": 30,
        }
      }
    >
      {text}
    </Text>
  );
}
```